packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -120,6 +216,11 @@ export const startPSM = async (
           value: makeRatio(GiveMintedFeeBP, minted, BASIS_POINTS),
         },
         MintLimit: { type: ParamTypes.AMOUNT, value: mintLimit },
+        ...oldState.governance,
```

Comment:
worth a callout comment that these are meant to override the values above. consider logging which keys will be overridden.